README.md

```diff
@@ -74,15 +74,17 @@ Total Flash memory (text + data + misc): 60554 bytes
 Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 ```
 
-It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
+It creates two binary images in the `BUILD/<TARGET_NAME>/<TOOLCHAIN_NAME>` directory. The original unpadded image is saved with the name `<project-name>_application.bin`. The padded bootloader image (used for combining with applications) is saved with the name `<project-name>.bin`.
 
-When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin`. This is the binary bootloader flashes and then jumps to.
+When the build succeeds, you have created a bootloader for your target. This bootloader defines the update binary to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin` on the sd card. This is the binary that the bootloader flashes and then jumps to.
```

Comment:
Can we just remove this first sentence? Could we clarify that the sd card must be formatted as FAT? Could we reword the second sentence to: "This bootloader looks for the update binary at `/mbed-os-example-bootolader-blinky_application.bin` on the attached sd card formatted with the FAT filesystem". When this bootloader finds this update binary, it flashes the new application and jumps to it.